utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -289,10 +289,13 @@ def _check_for_memleak(self):
                 memdiff = -1
                 if originalmem > 0:
                     memdiff = (finalmem - originalmem)/originalmem
-
+                tolerance = self._case.get_value("TEST_MEMLEAK_TOLERANCE")
+                if tolerance is None:
+                    tolerance = 0.1
+                expect(tolerance > 0.0 and tolerance < 1.0, "Bad value for memleak tolerance in test")
```

Comment:
I don't think this should necessarily be less than 1. What if we expect a 3x growth in memory, from 100MB to 300MB? Wouldn't that be given as a tolerance of 2.0?